core/src/main/java/software/amazon/awssdk/auth/profile/ProfileProperties.java

```diff
@@ -63,6 +64,5 @@
      */
     public static final String REGION = "region";
```

Comment:
Feels kinda wasteful to have a whole new class just for one constant - could this live somewhere else?

Reply:
There's a lot of constants here. The formatting was just perfect such that it's easy to miss the expand button!

Reply:
ahaha - nice.